html/images/lang/fr/labels/nature-et-progres.63x90.svg

```diff
@@ -0,0 +1,23 @@
+<svg width="65" height="90" viewBox="0 0 65 90" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
The filename should be nature-et-progres.65x90.svg

Reply:
done